README.md

```diff
@@ -2,45 +2,162 @@
 
-This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [public developer docs](https://azure.github.io/azure-sdk-for-cpp) or our versioned [developer docs](https://azure.github.io/azure-sdk-for-cpp).
+This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [public developer docs](https://azure.github.io/azure-sdk-for-cpp).
 
 ## Getting started
 
-To get started with a library, see the **README.md** file located in the library's project folder. You can find these library folders grouped by service in the `/sdk` directory.
+For the best development experience, we recommend that developers use the [CMake projects in Visual Studio](https://docs.microsoft.com/cpp/build/cmake-projects-in-visual-studio?view=vs-2019) to view and build the source code together with its dependencies. You can also use any other text editor of your choice, such as [VS Code](https://code.visualstudio.com/), along with the command line for building your application with the SDK.
 
-For tutorials, samples, quick starts, and other documentation, go to [Azure for C++ Developers](https://azure.github.io/azure-sdk-for-cpp).
+You can find additional information for specific libraries by navigating to the appropriate folder in the `/sdk` directory. See the **README.md** file located in the library's project folder. For example, [here is the doc for the storage client library](https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/storage#azure-storage-client-library-for-c).
 
-## Packages available
-Each service might have a number of libraries available from each of the following categories:
-* [Client - New Releases](#client-new-releases)
-* [Client - Previous Versions](#client-previous-versions)
+For API reference docs, tutorials, samples, quick starts, and other documentation, go to [Azure for C++ Developer Docs](https://azure.github.io/azure-sdk-for-cpp).
+
+### Download & Install the SDK
+
+The easiest way to get the C++ SDK is to leverage [vcpkg](https://github.com/microsoft/vcpkg). You'll need to have [Git](https://git-scm.com/downloads) installed before getting started.
+
+First clone and bootstrap vcpkg itself. You can install it anywhere on your machine, but **make note** of the directory where you clone the vcpkg repo.
+
+```cmd
+> git clone https://github.com/microsoft/vcpkg
+```
+
+On Windows:
+
+```cmd
+> .\vcpkg\bootstrap-vcpkg.bat
+```
+
+On Linux:
+
+```sh
+> ./vcpkg/bootstrap-vcpkg.sh
+```
+
+To install the libraries for your project, run the following, optionally specifying the triplet:
+
+```cmd
+> .\vcpkg\vcpkg install [packages to install]
+```
+
+For example, this will install the `x64-windows` triplet. On Windows, not specifying a triplet will default to x86-windows:

```

Comment:
"On Windows, not specifying a triplet will default to x86-windows" - are we sure?

Reply:
Yes. :)

Reply:
I am not questioning you saw it, I meant would this be the case on all customer Windows machines, i.e. is this behavior guaranteed/documented, or is it selected based on some system variables, and/or the default be changed by Vcpkg in the future? Is it possible to rewrite as less certain statement/avoid the statement about Vcpkg defaults?

Reply:
Yes, I had chatted with vcpkg folks and they mentioned the default is x86 which won't be changed.
If you don't have any specific vcpkg environment variables set (i.e. clean Windows machine), yes, that is the default.

I think it is important to call this out as it can be surprising to folks since they get hard to debug errors when trying to find the package and build their application. I don't see much benefit to try to make a less certain statement.

Here's a doc that mentions it:
https://vcpkg.readthedocs.io/en/latest/users/triplets/#additional-remarks